deploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash ./shell.nix
+
+set -eux
+
+PROJECT_NAME=bonk-api
+
+nix build .#packages.x86_64-linux.dockerImage
+skopeo \
+    --insecure-policy \
+    copy docker-archive:"$(realpath ./result)" \
+    docker://registry.fly.io/$PROJECT_NAME:latest \
+    --dest-creds x:"$(flyctl auth token)" \
```

Comment:
noooooooooo

Reply:
please send alternatives :sweat:

Reply:
`skopeo login` accepts a `--password-stdin` option, so
```
skopeo login docker://registry.fly.io --username x --password-stdin < <(flyctl auth token)
```
then this without `--dest-creds` should hopefully work

Reply:
Will fix in a follow-up shortly. Thanks!